src/main/java/com/eleks/academy/whoami/core/Turn.java

```diff
@@ -1,13 +1,17 @@
 package com.eleks.academy.whoami.core;
 
+import com.eleks.academy.whoami.core.action.PlayerAction;
+
 import java.util.List;
 
 public interface Turn {
-	
-	Player getGuesser();
-	
-	List<Player> getOtherPlayers();
 
-	void changeTurn();
+	void makeTurn(List<SynchronousPlayer> players, boolean samePlayer);
+
+	List<PlayerAction> getCurrentTurn();
+
+	List<List<PlayerAction>> getTurns();
+
+	void action(String player, PlayerAction action, String question);
```

Comment:
`String question` is better to `String value` you don't know if it is question or answer